src/mysky/utils.ts

```diff
@@ -64,3 +85,20 @@ export function popupCenter(url: string, winName: string, w: number, h: number):
   }
   return newWindow;
 }
+
+// TODO: Handle edge cases with specific servers as preferred portal?
+/**
+ * Returns whether we should redirect from the current portal to the preferred
+ * portal. The protocol prefixes are allowed to be different and there can be
+ * other differences like a trailing slash.
+ *
+ * @param currentPortalUrl - The current portal URL.
+ * @param preferredPortalUrl - The preferred portal URL.
+ * @returns - Whether the two URLs are equal for the purposes of redirecting.
+ */
+export function shouldRedirectToPreferredPortalUrl(currentPortalUrl: string, preferredPortalUrl: string): boolean {
+  currentPortalUrl = currentPortalUrl.split("//", 2)[1] || currentPortalUrl;
```

Comment:
`nit` I don't like the `split` on "//", it's just less correct even though the input is probably already sanitized in one way or the other
```
// strip protocol and trailing slash
currentPortalUrl = trimSuffix(currentPortalUrl.replace(/https:\/\/|http:\/\//, ''), "/");
preferredPortalUrl = trimSuffix(preferredPortalUrl.replace(/https:\/\/|http:\/\//, ''), "/");
return currentPortalUrl === preferredPortalUrl;
```

Reply:
Thanks, just needed a slight modification to make it case-insensitive.